flink-table/flink-sql-parser/src/test/java/org/apache/flink/sql/parser/FlinkSqlParserImplTest.java

```diff
@@ -687,6 +687,18 @@ public void testCastAsRowType() {
 			"CAST(`A` AS ROW(`F0` VARCHAR NOT NULL, `F1` TIMESTAMP) MULTISET)");
 	}
 
+	@Test
+	public void testValidateSqlCreateTable() {
+		String sql = "CREATE TABLE tbl1";
```

Comment:
testValidateCreateTableWithNakedTableName

Reply:
You mean the method name?  I've already changed it to testValidateEmpty...?

Reply:
You're right. I didn't notice that before. It's fixed now.